client/app/states/dashboard/dashboard.e2e.js

```diff
@@ -0,0 +1,20 @@
+/* eslint angular/log: 0 */
+/* eslint no-console: 0 */
+/* eslint no-undef: 0 */
```

Comment:
Thats definitely an option.  I was wondering if there was a cleaner way to deal with this.